code/__DEFINES/dcs/signals.dm

```diff
@@ -246,6 +247,8 @@
 #define COMSIG_CARBON_UNEQUIP_HAT "carbon_unequip_hat"				//from /mob/living/carbon/doUnEquip(obj/item/I, force, newloc, no_move, invdrop, silent)
 #define COMSIG_CARBON_EMBED_RIP "item_embed_start_rip"						// defined twice, in carbon and human's topics, fired when interacting with a valid embedded_object to pull it out (mob/living/carbon/target, /obj/item, /obj/item/bodypart/L)
 #define COMSIG_CARBON_EMBED_REMOVAL "item_embed_remove_safe"		// called when removing a given item from a mob, from mob/living/carbon/remove_embedded_object(mob/living/carbon/target, /obj/item)
+#define COMSIG_CARBON_WASHED "carbon_washed"						///from [/mob/living/carbon/washed]: ()
+	#define SPECIES_HATES_WATER 1									///flag to prevent nice shower moodlet
```

Comment:
```suggestion
#define SPECIES_HATES_WATER 1									///flag to prevent nice shower moodlet
```

Reply:
cursed github

Reply:
its indented like all the other flag returns